samples/vsphere/sso/external_psc_sso_workflow.py

```diff
@@ -82,7 +82,8 @@ def options(self):
         self.args = self.argparser.parse_args()
 
     def setup(self):
-        self.lswsdl = self.args.lswsdl
```

Comment:
Thanks @pgbidkar Looks good. You can remove the () on if. Also please squash the commits.